source/UIAHandler/_remoteOps/instructions.py

```diff
@@ -0,0 +1,758 @@
+# A part of NonVisual Desktop Access (NVDA)
+# This file is covered by the GNU General Public License.
+# See the file COPYING for more details.
+# Copyright (C) 2023-2023 NV Access Limited
+
+
+from __future__ import annotations
+from typing import cast
+from dataclasses import dataclass
+import ctypes
+from ctypes import POINTER
+from comtypes import GUID
+import UIAHandler
+from UIAHandler import UIA
+from . import lowLevel
+from . import builder
+from .builder import (
+	InstructionBase,
+)
+
+
+class _TypedInstruction(InstructionBase):
+
+	@property
+	def params(self):
+		return vars(self)
+
+
+@dataclass
+class Halt(_TypedInstruction):
+	opCode = lowLevel.InstructionType.Halt
+
+
+@dataclass
+class Fork(_TypedInstruction):
+	opCode = lowLevel.InstructionType.Fork
+	jumpTo: lowLevel.RelativeOffset
+
+
+@dataclass
+class ForkIfFalse(_TypedInstruction):
```

Comment:
Why is `ForkIfTrue` not implemented? Same for some other instructions. Would it be worth it to implement more instructions? (Good for new dev, or to spend a Sunday afternoon.)